practices/user_stories

```diff
@@ -0,0 +1,87 @@
+---
+slug: "user-stories"
+aspect: "architecture"
```

Comment:
Should it be a "processes", @pamichalik ?

Reply:
Yes, change it on processes, thx.